contents/docs/libraries/java/index.mdx

```diff
@@ -34,9 +34,9 @@ import JavaSendEvents from '../../integrate/send-events/_snippets/send-events-ja
 
 <JavaSendEvents />
 
-## Setting user properties
+## Person profiles and user properties
 
-To set [user properties](/docs/data/user-properties), include the properties you'd like to set when capturing an event: 
+By default, PostHog captures events with person profiles. To set [user properties](/docs/data/user-properties) in these profiles, include them when capturing an event:
```

Comment:
I wonder if for all these "by default" statements we should clarify it's for backwards compatibility. I'm concerned about the brand risk of offering a low event price but defaulting people into the high one, and I think it's probably better to over-communicate the reason behind it. We can also change the default later, but don't want to delay this release.

Reply:
cool, changed wording